### ISSUE TITLE
[Bug] Occasionally, under Mvis, it may suddenly ask to call the camera and go into an unresponsive state.

### ISSUE THREAD

**Issue by poly000** [EVAST9919/lazer-m-vis#9]
osu! 2020.801.0

![图片](https://user-images.githubusercontent.com/34085039/91628153-a9982f00-e9ef-11ea-9bbe-6ae147c9a407.png)

**Comment by EVAST9919** [comment]
Please, attach logs after crash happens

**Comment by Kashouryo** [comment]
Oh, you're using Huorong Security. This dialogue will also appear if an unsigned software tries to use your midi device.

**Comment by Kashouryo** [comment]
Oh, you're using Huorong Security. This dialogue will also appear if an unsigned software tries to use your midi device.

**Comment by EVAST9919** [comment]
Either way that's an osu-side issue (or may be even framework one), so there's nothing I can do on my end. You probably need to open the issue in osu-side.
Closing it here as such

**Comment by EVAST9919** [comment]
Just to be clear: I'm using only tools, provided by the framework itself, nothing more.

**Comment by poly000** [comment]
> 
> 
> Please, attach logs after crash happens

I'm sorry, I may not have access to the logs.
Because that's what I saw in the mf-osu group about a month ago.
The mfosu author also stated that no possible code for calling up the camera was found in your ruleset.
This issue is supposedly occurring occasionally, but it doesn't specify if it's occurring in Mvis.
I noticed that osu!.exe is not digitally signed, so I think it should be what Kashouryo is talking about.
